Vagrantfile

```diff
@@ -0,0 +1,27 @@
+Vagrant.configure("2") do |config|
```

Comment:
Hi,
Roger, I'll open another PR tonight to the `development` branch with only the
related file.

On Mon, Oct 19, 2015 at 10:46 am, Clochix < notifications@github.com [notifications@github.com] > wrote:
In Vagrantfile [https://github.com/cozy/cozy-home/pull/541#discussion_r42347782] :

> @@ -0,0 +1,27 @@
> +Vagrant.configure("2") do |config|


Please don't commit the Vagrantfile. Usually it should not be in your
application directory.

—
Reply to this email directly or view it on GitHub [https://github.com/cozy/cozy-home/pull/541/files#r42347782] .[https://github.com/notifications/beacon/AFSnBy1uEFmiTSNWrlEiyh-Td8gpmb-Cks5o9KVwgaJpZM4GQhHs.gif]